lesson_13/maps_ts/src/lesson13.ts

```diff
@@ -3,5 +3,22 @@
  * https://leetcode.com/problems/permutation-difference-between-two-strings
  */
 export function findPermutationDifference(s: string, t: string): number {
-  return 0;
-}
+  const letters = new Map<string,number>();
```

Comment:
See earlier code comments on variable name and commented code. Also, make sure to fix the format of this file.